Resources/public/js/views/services/ez-contentcreateviewservice.js

```diff
@@ -137,6 +137,15 @@ YUI.add('ez-contentcreateviewservice', function (Y) {
                 service.set('languageCode',e.selectedLanguageCode);
```

Comment:
actually, I realize that doing that in the callback of `destroy` is wrong because if you saved the version the callback is executed asynchronously. So instead, you should write something like:

```js
version.destroy({api: this.get('capi'), remove: true}, function (error) {
    if ( error ) {
        console.warn('Failed to remove the version ' + version.get('versionId'));
    }
});
this.set('version', new Y.eZ.Version());
view.set('languageCode', e.selectedLanguageCode);
this.fire('notify', {
// ...
```